projects/npm-tools/packages/npm-scripts/support/getFixturePath.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const path = require('path');
+
+function getFixturePath(...fixture) {
+	return path.join(__dirname, '../__fixtures__', ...fixture);
```

Comment:
While we're at it, should we change `'../__fixtures__'` to `'..', '__fixtures__'`?

I know it works (and tests on Windows in CI prove it), but maybe to be consistent?

Reply:
Yes. We should never use '/'  in any place, just `path.join` with multiple args. I learned that the hard way after several iterations in the bundler.

The root cause of the problem seems to be that we assume that we can convert paths from posix to windows by changing `/` and `\` when that's impossible because it is not a bijective mapping :-(.